src/core/SampleBuffer.cpp

```diff
@@ -362,16 +379,37 @@ void SampleBuffer::update(bool keepSettings)
 	}
 	Oscillator::generateAntiAliasUserWaveTable(this);
 
-	if (fileLoadError)
+	if (fileLoadError != None)
 	{
 		QString title = tr("Fail to open file");
-		QString message = tr("Audio files are limited to %1 MB "
-				"in size and %2 minutes of playing time"
-				).arg(fileSizeMax).arg(sampleLengthMax);
+		QString message;
+
+		m_audioFile = "";
```

Comment:
Could you explain what do you want to change with this additional line?

Reply:
The path to the invalid file gets saved into the project if it is not cleared.

Reply:
While this fixes the mentioned issue, it has an undesirable side effect. If the path to a valid file was saved, but the file failed to load for some reason, the path information will be lost.

Reply:
Removing it for now. Remaining problems with file loading from ``SampleBuffer`` can be discussed in #6287.